modules/ignitions/kubelet/files/kubelet-wrapper.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+# Wrapper script for launching kubelet.
+#
+# Make sure to set KUBELET_IMAGE_TAG to an image tag published here:
+# https://quay.io/repository/coreos/hyperkube?tab=tags Alternatively,
+# override KUBELET_IMAGE to a custom image.
+
+set -e
+
+function require_ev_all() {
+	for rev in $@ ; do
+		if [[ -z "${!rev}" ]]; then
+			echo "${rev}" is not set
+			exit 1
+		fi
+	done
+}
+
+function require_ev_one() {
+	for rev in $@ ; do
+		if [[ ! -z "${!rev}" ]]; then
+			return
+		fi
+	done
+	echo One of $@ must be set
+	exit 1
+}
+
+KUBELET_IMAGE_TAG="${KUBELET_IMAGE_TAG:-${KUBELET_VERSION}}"
+
+require_ev_one KUBELET_IMAGE KUBELET_IMAGE_TAG
+
+KUBELET_IMAGE_URL="${KUBELET_IMAGE_URL:-${KUBELET_ACI:-quay.io/coreos/hyperkube}}"
```

Comment:
this container image seems it isn't updated anymore, how about we choose another source as the default value?

Reply:
Yes, I will update it for making sure it's use an active repo.